### ISSUE TITLE
Contrast ratio is checked for elements with role="presentation"

### ISSUE THREAD

**Issue by wojtekmaj** [paypal/AATT#55]
Elements with role="presentation" should not be, in my opinion, checked for contrast ratio issues. By definition they are completely presentational, should be omitted by screen readers, don't contain any valuable content.
